examples/kotlin-multiplatform/build.gradle

```diff
@@ -17,8 +17,11 @@ kotlin {
     jvm {
         compilations.create('benchmark') { associateWith(compilations.main) }
     }
-    js('jsIr', IR) { nodejs() }
-    js('jsIrBuiltIn', IR) { nodejs() }
+    js(IR) {
+        nodejs()
+        compilations.create("defaultExecutor") { associateWith(compilations.main) }
```

Comment:
Minor: This compilation can be eliminated. Instead, the `js` target can be registered for benchmarking with the default executor.